internal/handler/task.go

```diff
@@ -46,29 +66,27 @@ func (h *Handler) GetCmd() *cobra.Command {
 	getCmd := &cobra.Command{
 		Use:   "get",
 		Short: "Get tasks by ID",
+		Args:  cobra.MinimumNArgs(1),
 		Run: func(cmd *cobra.Command, args []string) {
-			if len(args) < 1 {
-				log.Fatal(errors.New("task id is required"))
-			}
-			var ids []int
-			for _, i := range args {
-				i, err := strconv.Atoi(i)
-				if err != nil {
-					log.Fatal(err)
-				}
-				ids = append(ids, i)
+			ids, err := parseGet(args)
+			if err != nil {
+				log.Fatal(err)
 			}
-
 			for _, i := range ids {
 				t, err := h.service.GetTask(i)
 				if err != nil {
 					log.Fatal(err)
 				}
-				fmt.Println("Task:", t)
+				//check if task exists, because of nil pointer dereference
```

Comment:
We have to check, if task is not nil, because it will always throw "panic: runtime error: invalid memory address or nil pointer dereference"